src/main/java/in/org/projecteka/hiu/SecurityConfiguration.java

```diff
@@ -34,7 +36,6 @@
 
     private static final List<Map.Entry<String, HttpMethod>> SERVICE_ONLY_URLS = new ArrayList<>() {
         {
-            add(Map.entry("/consent/notification", HttpMethod.POST));
             add(Map.entry("/data/notification", HttpMethod.POST));
```

Comment:
is /data/notification API is called by gateway system?

Reply:
yes differentiate gateway and service urls